scripts/init.js

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2019 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+const fs = require('fs-extra')
```

Comment:
no need for this to be `fs-extra` and not `fs`, right?

Reply:
I suppose not, I just copied what was used before